doc/proposals/ansible-helm-addapi.md

```diff
@@ -0,0 +1,40 @@
+# Ansible/Helm add api command  Proposal for Operator SDK
+
+> Status: implementable
+> 
+
+- [Background](#background)
+- [Goals](#goals)
+- [Commands](#commands)
+- [Ansible_Changes] (#ansible changes)
+- [Helm_Changes] (#helm changes)
+
+## Background
+
+In current scenario, users are not able to add additional APIs for Ansible/Helm based opertators using SDK CLI.
+
+## Goals
+
+The goal of the proposal if implemented, is to enable users add additional APIs once the original Ansible/Helm operator is scaffolded.
+
+## Commands
+We are updating existing commands to accommodate the ansible/helm operator.  Changes to the `cmd\add\api.go` is needed.
+
+`operator-sdk add api --kind <kind> --api-version <group/version>`     
+
+## Ansible Changes
+
+* Udate ansible role at ../deplo/role.yaml
+* CRD and CR files are generated at ../deploy/crds/                                   
+* Update watches file at ../watches.yaml
+* Update playbook with the generated resources(TBD)
+
```

Comment:
Did you make a POC with Memcached Sample by adding manually all that is required to do check if will work well? 
Also, it has an option/flag that will do scaffold with watches playbooks, we need to check that as well.